assets/core/ingress-to-route-controller-clusterrolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:openshift-controller-manager:ingress-to-route-controller
+roleRef:
+  kind: ClusterRole
+  name: system:openshift:openshift-controller-manager:ingress-to-route-controller
+subjects:
+- kind: User
+  namespace: openshift-infra
+  name: system:route-controller-manager
```

Comment:
this should not be changed from the original and should keep binding to the `ingress-to-route-controller`

Reply:
wrt to the informers this should be handled by `route-controller-informer-clusterrole` for route-controller-manager

Reply:
I've deleted this manifest, because it was already applied as part of the openshift-router